invokeai/frontend/web/src/features/nodes/components/flow/Flow.tsx

```diff
@@ -149,10 +151,26 @@ export const Flow = () => {
     dispatch(contextMenusClosed());
   }, [dispatch]);
 
-  const onInit: OnInit = useCallback((flow) => {
-    $flow.set(flow);
-    flow.fitView();
-  }, []);
+  const onInit: OnInit = useCallback(
+    (flow) => {
+      $flow.set(flow);
+      flow.fitView();
+      flowWrapper.current = document.getElementById('workflow-editor');
```

Comment:
This `flowWrapper` ref can be provided directly to the `<ReactFlow />` component